ceph/nvmegw_cli/execute.py

```diff
@@ -6,7 +6,7 @@
 
 class ExecuteCommandMixin:
     BASE_CMD = "podman run --quiet --rm"
-    NVMEOF_CLI_IMAGE = "quay.io/ceph/nvmeof-cli:1.1.0"
+    NVMEOF_CLI_IMAGE = "quay.io/ceph/nvmeof-cli:latest"
```

Comment:
A separate commit for moving to `latest`. This allows to have meaningful history and also track why it was changed.

Reply:
Done.